infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/ExternalServerExposerStrategy.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.workspace.infrastructure.kubernetes.server;
+
+import io.fabric8.kubernetes.api.model.ServicePort;
+import java.util.Map;
+import org.eclipse.che.api.core.model.workspace.config.ServerConfig;
+import org.eclipse.che.workspace.infrastructure.kubernetes.environment.KubernetesEnvironment;
+
+/** @author Guy Daich */
+public interface ExternalServerExposerStrategy<T extends KubernetesEnvironment> {
+
+  public void exposeExternalServers(
```

Comment:
`public` access modifier is redundant here

Reply:
@sleshchenko Strategy is used by the ServerConverter, in another package (k8s.server), and also implemented by OS infra, so should be public.

Reply:
Sure it should be public. IDE shows me warning that it is redundant because `public` access modifier is default one for interfaces methods.